website/docs/cdktf/concepts/interoperability-workflows.html.md

```diff
@@ -1,19 +1,24 @@
 ---
 layout: "docs"
-page_title: "HCL & CDK for Terraform Interoperability"
-sidebar_current: "docs-home"
-description: "HCL and CDK for Terraform can interoperate"
+page_title: "HCL Interoperability"
+sidebar_current: "cdktf"
+description: "Learn how you can use configurations written in HCL and configurations written in CDK for Terraform together."
```

Comment:
```suggestion
description: "Use configurations written in HCL and configurations written in CDK for Terraform together to define and provision infrastructure."
```

slightly stronger for SEO to lead with the verb and avoid "learn to, etc."

Reply:
oh I didn't know that! I'll keep that in mind for the future - thank you!